packages/aws-cdk/lib/init-templates/v1/app/java/src/main/java/com/myorg/%name.PascalCased%App.template.java

```diff
@@ -2,14 +2,15 @@
 
 import software.amazon.awscdk.core.App;
 import software.amazon.awscdk.core.Environment;
+import software.amazon.awscdk.core.StackProps;
 
 import java.util.Arrays;
 
 public class %name.PascalCased%App {
     public static void main(final String[] args) {
         App app = new App();
 
-        %name.PascalCased%Stack.Builder.create(app, "%name.PascalCased%Stack")
+        new %name.PascalCased%Stack(app, "%name.PascalCased%Stack", StackProps.builder()
```

Comment:
Wouldn't this meant that the default generated app will not compile?

```java
new JavainitStack(app, "JavainitStack", StackProps.builder()).build();
```

I'd rather we add the code sample with instructions to the comment.

Reply:
removing all of the commented code results in
```java
        new JavaInitStack(app, "JavaInitStack", StackProps.builder().build());
 ```
 which does compile.

I'm fine with also just making separate examples as comments I'll move to that.

Reply:
Im afraid this will become a CDK idiom and will be copied paste to infinity by users assuming that passing the empty `StackProps.builder().build()` to the ctor has some effect.